rework_pysatl_mpest/preprocessing/components_family/criterions/frequency_criterions.py

```diff
@@ -0,0 +1,279 @@
+"""Module which contains frequency features (from sound recognition) for a mixture classifier"""
+
+__author__ = "Mark Dubrovchenko"
+__copyright__ = "Copyright (c) 2025 PySATL project"
+__license__ = "SPDX-License-Identifier: MIT"
+
+import numpy as np
+from pywt import wavedec
+from rework_pysatl_mpest.preprocessing.components_family.criterions.abstract_criterion import (
+    AHistClassifierCriterion,
+)
+from scipy.fft import rfft
+from scipy.fftpack import dct
+from scipy.signal import periodogram
+
+
+class CDct(AHistClassifierCriterion):
+    def __init__(self, dct_type: int) -> None:
+        self.dct_type = dct_type
+
+    @property
+    def name(self) -> str:
+        return f"DCT C{self.dct_type} Criterion"
+
+    def score(self, hist: np.ndarray) -> float:
```

Comment:
For consistency with other modules, it would be better to use `ArrayLike` for type hinting and `np.asarray()` for conversion here.
```suggestion
    def score(self, hist: ArrayLike) -> float:
        hist = np.asarray(X)
```